src/clusterfuzz/_internal/bot/tasks/utasks/uworker_io.py

```diff
@@ -469,29 +476,6 @@ class RegressionTaskInput(UworkerInput):
   PROTO_CLS = uworker_msg_pb2.RegressionTaskInput
 
 
-class AnalyzeTaskOutput(UworkerMsg):  # pylint: disable=abstract-method
```

Comment:
I guess I already did this with input but I think the model for base inputs and specific inputs are confused. In protos theyre subfields, here they are child objects.

Reply:
There is confusion indeed. I think that to avoid code duplication, and since normally TaskOutput types should behave similarly, maybe we could create a `TaskOutput` class and make Analyze,Minimize,... extend it. I made this change because I saw we were already duplicating the `save_rich_type` method in `MinimizeTaskOutput` and `AnalyzeTaskOutput`.
Same thing should probably be done for TaskInputs instead of extending `UworkerInput`.
WDYT?

Reply:
Eh I think inheriting from UworkerOutput and UworkerInput is fine.